scimath/Fitting/FitGaussian.tcc

```diff
@@ -440,9 +441,10 @@ Matrix<T> FitGaussian<T>::fit(const Matrix<T>& pos, const Vector<T>& f,
             //best fit so far - write parameters to solution matrix
             for (uInt g = 0; g < itsNGaussians; g++) {  
               for (uInt p = 0; p < ngpars; p++) {
-                solutionparameters(g,p) = solution(g*ngpars+p);
+                itsSolutionParameters(g,p) = solution(g*ngpars+p);
               }
             }
+            itsSolutionErrors = errors();
```

Comment:
I don't understand since errors() returns itsSolutionErrors, thus assigns to itself.
It looks as if the errors are not set.

Reply:
Yup - my fault - I meant to access the getErrors method from the fitter.

Reply:
I'm about to wrap up for the day - but I'll fix up both of these issues asap. Thx